11-Implementing_combineReducers()_from_Scratch.md

```diff
@@ -2,17 +2,17 @@
 
 Now that we know how to use `combineReducers()` to save us some time, let's implement it from scratch in order to understand it deeper.
 
-Recall that the only argument to `combineReducers()` is an object with a map to our reducers, so we'll start by writing a function that accepts a parameter we'll call "reducers".
+Recall that the only argument to `combineReducers()` is a function declaration to map through our reducers, so we'll start by writing a function that accepts a parameter we'll call "reducers".
 
 Since `combineReducers()` returns a reducer function, it must have the signature of a reducer function (the state and an action).
 
-Inside of our return reducer, we call the `Object.keys()` function to get all the keys from our `reducers` object (in our example, this is `todos` and `visibilityFilter`). 
+Inside of our return reducer, we call the `Object.keys()` function to get all the keys from our `reducers` object (in our example, this is `todos` and `visibilityFilter`).
 
-We then run `reduce()` on the keys because we want to produce a single value that represents the next state. We do this by iterating over every key and running its associated reducer.
+We then run `reduce()` on the keys because we want to produce a single value that represents the next state. We do this by accumulating over every key and running its associated reducer.
```

Comment:
well since it is a reduce function it is literally an accumulator, but most importantly, dan's words not mine ;-)